azext_iot/tests/test_iot_central_int.py

```diff
@@ -170,59 +170,6 @@ def test_central_device_registration_info(self):
         assert json_result["device_registration_info"] is not None
```

Comment:
I'd suggest writing a UT to ensure theres no regression around having extraneous information in the result